src/client/pages/instance/settings.vue

```diff
@@ -135,6 +135,18 @@
 		</div>
 	</section>
 
+	<section class="_card _vMargin">
```

Comment:
こうではない？最新版マージしたときに対応抜けてそう
```suggestion
	<section class="_section">
```